backend/status_codes.md

```diff
@@ -0,0 +1,53 @@
+# Status Codes
+
+Common cases
+--------------
+
+Status codes for common scenarios:
+
+* General:
```

Comment:
The data type errors covered by schema returns `400 bad request`. We are using `precondition failed` when the request failed with some business validation